content/08-mongodb/11-mongodb-atlas-setup.mdx

```diff
@@ -0,0 +1,136 @@
+---
+title: 'Introduction to provisioning MongoDB Atlas'
+metaTitle: "MongoDB Atlas Setup | Prisma's Data Guide"
+metaDescription: "In this guide, we'll cover provisioning MongoDB Atlas."
+metaImage: '/content/mongodb/header.png'
+authors: ['alexemerich']
+---
+## Introduction
+
+MongoDB Atlas is MongoDB's cloud database service. It offers data distribution and mobility across major cloud providers AWS, Azure, and Google Gloud. It is popular for its built-in automation for resource and workload optimization. 
+
+In this guide, we are going to walk through the steps to provision a MongoDB Atlas cluster from the browser. We'll walk through all the options to configuring a cluster up until deploy.
```

Comment:
```suggestion
In this guide, we are going to walk through the steps to provision a MongoDB Atlas cluster from the browser. We'll walk through all of the options from configuring a cluster up until deploy.
```

Maybe? Or maybe switch the "from" to "to" again and change "until" to "through"?  It depends on your intent mainly.